t/t2801-top-cmd.t

```diff
@@ -32,6 +32,12 @@ job_list_wait_state() {
 	return 0
 }
 
+# flux-top will redraw panes when a heartbeat is received, which could
+# lead to racy output with tests below.  Set the heartbeat period to
+# something really high to avoid it.
+test_expect_success 'set high heartbeat period' '
+	flux module reload heartbeat period=30m
+'
```

Comment:
Question: what happens if you just unload `heartbeat`?

Reply:
ahh i guess that would work too.  I initially thought `flux_event_subscribe()` would fail if the event didn't actually exist, but that's stupid of course, it's an event stream not an rpc.  Do think removing preferable to setting a long period?

Reply:
Eh, I was more just curious. If it worked to remove it then that seems more clear (do not ever generate a heartbeat)

Reply:
well hopefully it works, since the race is pretty rare, hard to verify :P